Foundation/CGFloat.swift

```diff
@@ -143,6 +143,10 @@ public struct CGFloat {
     
     /// The native value.
     public var native: NativeType
+
+    @_transparent public var double: Double {
+        return Double(native)
+    }
```

Comment:
This still doesn't feel right to me for two reasons:

1. I don't like phrasing this as a type conversion hidden as a property.
2. It cannot be public.
3. It feels like a separate issue than about Android, and probably should be its own PR so we can figure it out independently.